applications/ThermalDEMApplication/custom_elements/thermal_spheric_particle.h

```diff
@@ -229,9 +229,17 @@ namespace Kratos
       double mTotalHeatFlux;
 
       // Energy properties
-      double mPreviousViscodampingEnergy; // accumulated energy dissipation from previous interaction:   viscodamping 
-      double mPreviousFrictionalEnergy;   // accumulated energy dissipation from previous interaction:   frictional
-      double mPreviousRollResistEnergy;   // accumulated energy dissipation from previous interaction:   rolling resistance
+      double mPreviousViscodampingEnergy; // accumulated energy dissipation from previous interaction: viscodamping 
+      double mPreviousFrictionalEnergy;   // accumulated energy dissipation from previous interaction: frictional
+      double mPreviousRollResistEnergy;   // accumulated energy dissipation from previous interaction: rolling resistance
+
+      // Heat maps
+      std::vector<std::vector<std::vector<double>>> mHeatMapGenerationDampingPP;  // Local heat map matrix for heat generaion by damping between particle-particle
```

Comment:
@loumalouomega can we create a 3d Matrix ? 

I mean Matrix rA(3,3,3); for example